app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -32,5 +73,40 @@ def provider_relationship_permissions_needing_setup
     def require_manage_organisations_permission!
       render_404 unless current_provider_user.authorisation.can_manage_organisations_for_at_least_one_provider?
     end
+
+    def organisation_permissions_wizard_store
+      key = "organisation_permissions_wizard_store_#{current_provider_user.id}"
+      WizardStateStores::RedisStore.new(key: key)
+    end
+
+    def current_relationship_description
+      ProviderRelationshipPermissionAsProviderUserPresenter.new(@current_relationship, current_provider_user).provider_relationship_description
+    end
+
+    helper_method :current_relationship_description
+
+    def previous_page_path(wizard)
```

Comment:
Look into the wizard history tooling instead of mapping the paths here

Reply:
Decided in person to keep this as-is. 

The path history module is used to save the list of wizard steps previously accessed, so we'd still need this path logic in the controller. Since this is quite a simple wizard, I think it's more clear to just write down the next step and previous step logic in the wizard without hiding it in the path history